policies/aws_s3_policies/aws_s3_bucket_public_read.py

```diff
@@ -8,6 +8,9 @@
 
 
 def policy(resource):
+    if "IsPublic" in resource and resource["IsPublic"]:
```

Comment:
Hi @calkim-panther, just wanted your insight here.

Currently, this `IsPublic` flag is [determined by AWS](https://docs.aws.amazon.com/AmazonS3/latest/userguide/access-control-block-public-access.html#access-control-block-public-access-policy-status), however `IsPublic` could mean that the bucket could be open to updates, creation, deletion, or reading. 

Do you think this change belongs here? I'm wondering whether it should be in it's own policy, since `IsPublic` means more than it just being a publicly readable s3 bucket.